.gitattributes

```diff
@@ -0,0 +1 @@
+Cargo.lock binary
```

Comment:
Do we really want this? Seems like it prevents `Cargo.lock` from being merged automatically but I can't say I have had an issue with that (I believe the lockfile format was actually tweaked to make it easier to merge).

https://stackoverflow.com/questions/33085428/gitattributes-and-the-binary-option

Reply:
Yes. We do this in our other projects, though we also give it a merge strategy, usually `merge=theirs`. I wanted to experiment with not including the merge strategy, because that seems to have different semantics on rebase, so which is "theirs" and "ours" is awkward (git is...awkward).